test/unit/language-extensions/multi.spec.ts

```diff
@@ -24,6 +24,30 @@ test("multi example use case", () => {
         .expectToEqual({ a: "foo", b: 5 });
 });
 
+// https://github.com/TypeScriptToLua/TypeScriptToLua/issues/995
+test("Destructuring assignment of LuaMultiReturn", () => {
+    util.testModule`
+        function multiReturn(): LuaMultiReturn<[number, number, number]> {
+            return $multi(1, 2, 3);
+        }
+
+        const [a, ...b] = multiReturn();
+        export {a, b};
+    `
+        .setOptions(multiProjectOptions)
+        .expectToEqual({ a: 1, b: [2, 3] });
+    util.testModule`
```

Comment:
Tests should only test one thing, could you split this second testModule into a second `test("Destructuring assignment of LuaMultiReturn returning nil", () => {`